src/muxer.ts

```diff
@@ -30,37 +30,35 @@ export class DataChannelMuxer implements StreamMuxer {
   source: Source<Uint8Array> = nopSource;
   sink: Sink<Uint8Array, Promise<void>> = nopSink;
 
-
-  constructor(peerConnection: RTCPeerConnection, init?: StreamMuxerInit) {
+  constructor (peerConnection: RTCPeerConnection, init?: StreamMuxerInit) {
     this.init = init
     this.peerConnection = peerConnection
-    this.peerConnection.ondatachannel = ({channel}) => {
+    this.peerConnection.ondatachannel = ({ channel }) => {
       const stream = new WebRTCStream({
         channel,
         stat: {
           direction: 'inbound',
           timeline: {
-            open: 0,
+            open: 0
           }
         },
         closeCb: init?.onStreamEnd
       })
-      if (init?.onIncomingStream) {
-        init.onIncomingStream!(stream)
+      if ((init?.onIncomingStream) != null) {
```

Comment:
did the linter ask this? Why the need to check explicit or `null`? Doesn't `init?.onIncomingStream` already do this for you?

Reply:
The linter asked for this and I don't remember why.

Reply:
Might want to check why, as there are a bunch of changes like this, which I am not entirely sure why they would be better....

Reply:
In general, the linter didn't like these type coercions and wanted explicit types.  When I turn this on, I get:
```
Unexpected nullable string value in conditional. Please handle the nullish/empty cases explicitly. [Error/@typescript-eslint/strict-boolean-expressions]
```